pages/common/exo-compute-block-storage.md

```diff
@@ -0,0 +1,32 @@
+# exo compute block-storage
+
+> CLI for managing the Exoscale Block Storage service.
```

Comment:
```suggestion
> Manage the Exoscale Block Storage service.
```
See the style guide https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#program-description